addon/v1alpha1/types_addondeploymentconfig.go

```diff
@@ -50,6 +50,10 @@ type AddOnDeploymentConfigSpec struct {
 	// Empty means no proxy settings is available.
 	// +optional
 	ProxyConfig ProxyConfig `json:"proxyConfig,omitempty"`
+
+	// AgentInstallNamespace is the namespace where the add-on agent should be installed on the managed cluster.
+	// +optional
+	AgentInstallNamespace string `json:"agentInstallNamespace,omitempty"`
```

Comment:
should we add a default value e.g. open-cluster-management-agent-addon?

Reply:
ok, fixed